pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -562,12 +611,131 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 				task.Namespace, task.Name, node.Name, fit)
 		}
 
+		// Check VolumeBinding
+		if predicate.volumeBindingEnable {
+			isSkipVolumeBinding := handleSkipPredicatePlugin(task, skipPlugins, volumeBindingPlugin.Name(), node)
+			if !isSkipVolumeBinding {
+				status := pp.volumeBindingPlugin.Filter(context.TODO(), state, task.Pod, nodeInfo)
+				volumeBindingStatus := api.ConvertPredicateStatus(status)
+				if volumeBindingStatus.Code != api.Success {
+					predicateStatus = append(predicateStatus, volumeBindingStatus)
+					if ShouldAbort(volumeBindingStatus) {
+						return api.NewFitErrWithStatus(task, node, predicateStatus...)
+					}
+				}
+			}
+		}
+
 		if len(predicateStatus) > 0 {
 			return api.NewFitErrWithStatus(task, node, predicateStatus...)
 		}
 
 		return nil
 	})
+
+	ssn.AddBatchNodeOrderFn(pp.Name(), func(task *api.TaskInfo, nodes []*api.NodeInfo) (map[string]float64, error) {
```

Comment:
We'd better add a comment here, because nodeorder also has a BatchNodeorderFn, which is duplicated and should be normalized to predicates plugin, and it's a TODO here.

Reply:
I have added it